### PR TITLE
Update bug_report.md

### PR DESCRIPTION
**Pull request by BulatKha**
Added:
1. Actual result, 
2. Suggestions on naming convention
3. CAPSed the system info partially and moved to top. Alternative: use the warning emoji and move outside the commented block
4. Specified few examples of additional context, that might help collecting more info n the bug.

- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] No need to
- [ ] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by BulatKha** [comment]
@Haarolean changed it a bit. Please re-check

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -50,5 +62,9 @@ Steps to reproduce the behavior:
 
 **Additional context**
 <!--
-(Add any other context about the problem here)
+Add any other context about the problem here. E.g.: 
+1. Additional checks done with outcome, if that can help understand the issue better or pinpoint the root cause
```
Comment (Haarolean) [inline .github/ISSUE_TEMPLATE/bug_report.md:66]: Are there alternative scenarios (different input data, different configuration/setup) in which the problem does (not) occur? If yes, please provide the steps for these scenarios.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -50,5 +62,9 @@ Steps to reproduce the behavior:
...
+2. Related issues
```
Comment (Haarolean) [inline .github/ISSUE_TEMPLATE/bug_report.md:67]:
```suggestion
2. Related issues (if there are any)
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -50,5 +62,9 @@ Steps to reproduce the behavior:
 
 **Additional context**
 <!--
-(Add any other context about the problem here)
+Add any other context about the problem here. E.g.: 
+1. Additional checks done with outcome, if that can help understand the issue better or pinpoint the root cause
+2. Related issues 
+3. Logs (if available)
+4. Impact on the end user in general.
```
Comment (Haarolean) [inline .github/ISSUE_TEMPLATE/bug_report.md:69]: Is there any non-obvious/serious behaviour/effect on the end-user with this issue?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -15,18 +15,25 @@ https://github.com/provectus/kafka-ui/discussions
 
 -->
 
+<!--
+Please follow the naming conventions for bugs:
+<Feature> :  <Compact, but specific problem summary>
```
Comment (Haarolean) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]:
```suggestion
<Feature/Area/Scope> :  <Compact, but specific problem summary> 
```

Comment (BulatKha) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (BulatKha) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (BulatKha) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (BulatKha) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done

**Review by Haarolean — APPROVED** [review]

**Pull request merged**